src/librbd/journal/Types.h

```diff
@@ -253,6 +254,18 @@ struct FlattenEvent : public OpEventBase {
   using OpEventBase::dump;
 };
 
+struct ExclusiveLockReleaseEvent : public OpEventBase {
```

Comment:
Don't need to inherit from OpEventBase -- this isn't a maintenance operation.  As a result, you won't need an op_tid.